app/main/mutations.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Hi</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="./reset.css">
+</head>
+<div id="root">
+    <!-- React goes here -->
+</div>
+<footer>
+    <script>
+        const render = require("../.compiled/renderer.bundle.js").default.mutations;
+        render();
+
+        // Add React dev tools in development. 
+        if (process.env.NODE_ENV !== 'production') {
```

Comment:
the NODE_ENV for production is "prod". That's at least how I have it set up so we should stay consistent.